L1Trigger/TrackFindingTracklet/interface/Constants.h

```diff
@@ -0,0 +1,671 @@
+#ifndef CONSTANTS_H
+#define CONSTANTS_H
+
+// ------------------------------------------------------------------------------------
+// Different options for root output (imath), tracklet vs hybrid, HLS for KF
+// ------------------------------------------------------------------------------------
+
+//Uncomment if you want root output
+//#define USEROOT
+
+//Comment out to run the tracklet-only algorithm
+#ifdef CMSSW_GIT_HASH
+#define USEHYBRID
+#endif
+
+//Uncomment to use HLS version of KF. Also read TrackFindingTMTT/README_HLS.txt
+//#ifdef USEHYBRID
+//#define USE_HLS
+//#endif
+
+//Uncomment to run the HLS version of the KF if using the Hybrid (instead of the C++ KF).
+//(Please also follow the instructions in L1Trigger/TrackFindingTMTT/README_HLS.txt).
+//#define USE_HLS
+
+// ------------------------------------------------------------------------------------
+// Turn on/off debug info
+// ------------------------------------------------------------------------------------
+
+static const bool printDebugKF = false;  // if true print lots of debugging statements related to the KF fit
+static const bool debug1 = false;        // print detailed debug information about tracklet tracking
```

Comment:
shouldn't this be in some namespace to avoid potential duplicate symbols?

Reply:
namespace introduced